src/utils/database.js

```diff
@@ -37,7 +37,7 @@ export const closeDatabase = async () => {
 
 export const clearDatabase = async () => {
   const { collections } = mongoose.connection;
-  Object.keys(collections).forEach(collection =>
-    mongoose.connection.collection(collection).deleteMany({}),
+  Object.keys(collections).forEach( async collection =>
```

Comment:
O good catch i forgot to add await when refactoring! I think we dont need async on this line since we have it on the top

Reply:
ah actually we should keep async here and delete async on the top!